.github/workflows/test-fork-prs.yml

```diff
@@ -201,6 +201,36 @@ jobs:
       commit: ${{ needs.setup.outputs.commit_id }}
       repository: ${{ github.repository }}
 
+  license-checker:
+    runs-on: ubuntu-latest
+    needs: setup-cache
+    steps:
+      - name: Checkout Amplify UI
+        uses: actions/checkout@24cb9080177205b6e8c946b17badbe402adc938f # v3.4.0 https://github.com/actions/checkout/commit/24cb9080177205b6e8c946b17badbe402adc938f
+        with:
+          ref: ${{ inputs.commit }}
+          repository: ${{ inputs.repository }}
```

Comment:
```suggestion
```

For internal PRs, `checkout` is smart enough to infer ref and repository (and `inputs` only exist for reusable workflows).